api/taskcluster/webhook_test.go

```diff
@@ -176,7 +176,6 @@ func TestCreateAllRuns_one_error(t *testing.T) {
 	suitesAPI := checks.NewMockSuitesAPI(mockC)
 	suite := shared.CheckSuite{SHA: sha}
 	suitesAPI.EXPECT().GetSuitesForSHA(sha).Return([]shared.CheckSuite{suite}, nil)
-	suitesAPI.EXPECT().PendingCheckRun(suite, sharedtest.SameProductSpec("chrome"))
 	suitesAPI.EXPECT().PendingCheckRun(suite, sharedtest.SameProductSpec("firefox"))
```

Comment:
Sadly, this is non-deterministic. The mock server is set up to respond OK to the first request, but we don't know which goroutine runs first and hence we can't guarantee whether the first one would be chrome or firefox. I think it's a reasonable compromise to use `gomock.Any` here.

Reply:
```suggestion
	suitesAPI.EXPECT().PendingCheckRun(suite, gomock.Any())
```